sdk/jobs/automl-standalone-jobs/automl-image-classification-multilabel-task-fridge-items/automl-image-classification-multilabel-task-fridge-items.ipynb

```diff
@@ -204,6 +286,44 @@
     "# my_validation_data_input = Input(type=AssetTypes.MLTABLE, path=\"azureml://datastores/workspaceblobstore/paths/vision-classification/valid\")"
    ]
   },
+  {
+   "cell_type": "markdown",
```

Comment:
Could you also please add the text equivalent to "Configuring your AutoML run for image tasks" from v1 notebooks here. I think it has useful links to the documentation and is good for the flow of the notebook.

Reply:
Please note that the section of "Using default hyperparameter values for the specified algorithm" with code and text will be added as part of another task. So, you need not add it.